Description: Upstream changes introduced in version 0.60.6-5
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 aspell (0.60.6-5) UNRELEASED; urgency=low
 .
   * Added a grammatical fix for aspell.1 to 05_doc_fix.diff
     (Closes: #576749)
 .
   * debian/control: added Vcs fields for git repository
 .
   * Added new patch 06_null_input_segfault_fix.diff
     (Closes: #596642)
 .
   * debian/rules: changed pkgdatadir to /usr/share/aspell which is a more
     appropriate location for that data than /usr/lib/aspell, and leave the
     spell and ispell scripts there so that run-with-aspell works
     (Closes: #599653)
 .
 The person named in the Author field signed this changelog entry.
Author: Brian Nelson <pyro@debian.org>
Bug-Debian: http://bugs.debian.org/576749
Bug-Debian: http://bugs.debian.org/596642
Bug-Debian: http://bugs.debian.org/599653

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- aspell-0.60.6.orig/prog/check_funs.cpp
+++ aspell-0.60.6/prog/check_funs.cpp
@@ -441,7 +441,7 @@ void get_choice(int & c) {
       c0 = wgetch(choice_w);
     } while (c0 == ERR);
     if (c0 == KEY_BREAK)
-      c0 = control('c');
+      c = control('c');
     if (1 <= c0 && c0 < 128) {
       c = static_cast<char>(c0);
       waddch(choice_w,c);
